vuepress/docs/next/docs/curate/troubleshooting-guide.md

```diff
@@ -5,12 +5,11 @@
 
 Currently the Entando Component Manager logs are only available in Kubernetes through Openshift dashboards or CLI tools like K9s and kubectl.
 
-
 ### Solution
-If your Entando cluster has more than one Entando App custom resource, find the component manager in question using the Entando App name and namespace where the app has been deployed. Assuming you have a single Entando App named `quickstart` and your cluster namespace is called `YOUR-NAMESPACE`, 
+If your Entando cluster has more than one Entando App custom resource, find the component manager in question using the EntandoApp name in the namespace where the app has been deployed. Assuming you have a single EntandoApp named `quickstart`, 
```

Comment:
why is the first entando app not one word?

"find the component manager in question" is a little wordy and imprecise (the CM is not directly discussed here, nor is it under investigation or a source of controversy). find the applicable component manager? correct, appropriate, etc...

Reply:
this alerted me to the fact that the command below is not the right one. It actually should be the cm deployment. Reworded